src/commands/internal/create-changelog.ts

```diff
@@ -101,26 +112,51 @@ export async function getChangelogText(mode: string, startRef: string): Promise<
     process.exit(2);
   }
 
+  const previousPreVersions = getPreviousPreVersionsInReleaseChannel(nextVersion);
+  const shouldShowPreviousPreVersions = previousPreVersions.length > 0;
+  let currentPreviousPreVersion = shouldShowPreviousPreVersions ? { tag: nextVersion, date: new Date() } : null;
+  let currentPreviousPreVersionIndex = -1;
+
   const mergedPullRequestsText = mergedPullRequests
     .map((pr: PullRequest): string => {
-      const mergedAt = moment(pr.mergedAt).format('YYYY-MM-DD');
+      let text = '';
+      const mergedAtDate = moment(pr.mergedAt);
+      const mergedAtString = mergedAtDate.format('YYYY-MM-DD');
+      const introducedInCurrentPreviousPreVersion =
+        currentPreviousPreVersion != null && mergedAtDate.isBefore(currentPreviousPreVersion.date);
+
+      if (introducedInCurrentPreviousPreVersion) {
+        const header =
+          currentPreviousPreVersionIndex === -1
+            ? `${currentPreviousPreVersion.tag} (this release)`
+            : `[${currentPreviousPreVersion.tag}](https://github.com/${GITHUB_REPO}/releases/tag/${currentPreviousPreVersion.tag})`;
```

Comment:
Gibt es einen Grund, dass für die aktuelle Version kein Link hinterlegt wird?

```suggestion
          currentPreviousPreVersionIndex === -1
            ? `[${currentPreviousPreVersion.tag} (this release)](https://github.com/${GITHUB_REPO}/releases/tag/${currentPreviousPreVersion.tag})`
            : `[${currentPreviousPreVersion.tag}](https://github.com/${GITHUB_REPO}/releases/tag/${currentPreviousPreVersion.tag})`;
```

Reply:
Weil du auf dem Release bist, das du hier verlinken würdest ;-)

Reply:
Ah! Ich war grad irgendwie bei Slack-Announcern und dachte, dass es doch sinnvoll wäre, wenn der Link zum aktuellen Release auf GitHub auch da wäre. Aber wir sind natürlich schon auf GitHub. Dann passt das natürlich 🙂